deparse.c

```diff
@@ -2453,6 +2529,36 @@ sqlite_deparse_expr(Expr *node, deparse_expr_cxt *context)
 	}
 }
 
+/*
+ * Preferred SQLite affinity from "column_type" foreign column option
+ * SQLITE_NULL if no value or no normal value
+ */
+int
```

Comment:
please add "static" before "int preferred_sqlite_affinity"

Reply:
Please verify, @lamdn1409 !

Reply:
I confirmed the change.